nilearn/reporting/data/html/report_body_template.html

```diff
@@ -140,7 +140,7 @@
   font-size: 12pt;
   font-weight: 200;
   border-radius: 6px;
-  padding: 5px;css=resource_path.joinpath('css'),
```

Comment:
this seems to have been there for a long time and I cannot figure out what it was intended to do but it for sure is not proper css.